yarn/datadog_checks/yarn/data/conf.yaml.example

```diff
@@ -102,6 +102,11 @@ instances:
     #
     # collect_app_metrics: true
 
+    ## @param collect_node_metrics - boolean - optional - default: true
```

Comment:
Let's add this option to the configuration spec https://github.com/DataDog/integrations-core/blob/master/yarn/assets/configuration/spec.yaml#L68-L72

You can then run `ddev validate config -s` which will sync and update the conf.yaml.example (standardized formatting)

Reply:
Added the option to `configuration/spec.yaml`, but `ddev` has never worked for me.  In theory it should generate the same `conf.yaml.example` though